profiles/killbill/src/test/java/org/killbill/billing/jaxrs/TestPayment.java

```diff
@@ -442,6 +443,24 @@ public void testComboAuthorization() throws Exception {
     }
 
     @Test(groups = "slow")
+    public void testComboAuthorizationAbortedPayment() throws Exception {
+        final Account accountJson = getAccount();
+        accountJson.setAccountId(null);
+        final String paymentExternalKey = UUID.randomUUID().toString();
+        final ComboPaymentTransaction comboPaymentTransaction = createComboPaymentTransaction(accountJson, paymentExternalKey);
+
+        mockPaymentControlProviderPlugin.setAborted(true);
+        try {
+            killBillClient.createPayment(comboPaymentTransaction, Arrays.asList(MockPaymentControlProviderPlugin.PLUGIN_NAME), ImmutableMap.<String, String>of(), createdBy, reason, comment);
+            fail();
+        } catch (KillBillClientException e) {
+            assertEquals(e.getResponse().getStatusCode(), 422);
```

Comment:
Can you use `ErrorCode. PAYMENT_PLUGIN_API_ABORTED.getCode()` instead of numeral value (those sometimes change across major release.